src/commonmark.rs

```diff
@@ -440,6 +440,68 @@ impl<'a, W: Write + 'a> Context<'a, W> {
         );
         write!(self.output.writer, "{}", osc(&command))
     }
+
+    /// Write an inline image for Terminology.
```

Comment:
I think this method needs much more documentation; I don't even understand what it does at all 🙈 

Can you perhaps explain how it works, and also how the image format of terminology looks like?